khmer/_oxli/graphs.pxd

```diff
@@ -85,6 +85,43 @@ cdef extern from "oxli/hashtable.hh" namespace "oxli":
     cdef cppclass CpNodetable "oxli::Nodetable" (CpHashtable):
         CpNodetable(WordLength, vector[uint64_t])
 
+    cdef cppclass CpQFCounttable "oxli::QFCounttable":
```

Comment:
Tried that and you end up with a bunch of these:
```
khmer/_oxli/graphs.pyx:58:43: Cannot assign type 'char *' to 'HashIntoType'

Error compiling Cython file:
------------------------------------------------------------
...
        For Nodetables and Counttables, this function will fail if the
        supplied k-mer contains non-ACGT characters.
        """
        if isinstance(kmer, str):
            temp = kmer.encode('utf-8')
            return deref(self.c_table).get_count(<char*>temp)
```

not sure why defining them again in the same order (instead of inheriting) fixes this :-/